rocks/RocksDatabase.java

```diff
@@ -329,6 +377,90 @@ public void delete() {
         }
     }
 
+    /**
+     * WARNING: we can break backward compatibility or corrupt user data by changing these options and using them
+     * with existing databases
+     */
+    private org.rocksdb.Options rocksSchemaOptions() {
+        return new org.rocksdb.Options()
+                .setCreateIfMissing(true)
+                .setCompressionType(NO_COMPRESSION)
+                .setTableFormatConfig(rocksSchemaTableOptions());
+    }
+
+    private BlockBasedTableConfig rocksSchemaTableOptions() {
+        BlockBasedTableConfig rocksDBTableOptions = new BlockBasedTableConfig();
+        LRUCache uncompressedCache = new LRUCache(64 * MB);
+        rocksDBTableOptions.setBlockSize(16 * KB);
+        rocksDBTableOptions.setFormatVersion(5);
+        rocksDBTableOptions.setIndexBlockRestartInterval(16);
+        rocksDBTableOptions.setEnableIndexCompression(false);
+        rocksDBTableOptions.setBlockCache(uncompressedCache);
+        return rocksDBTableOptions;
+    }
```

Comment:
we should configure the Schema database differently to the Data database - high level, we don't need any bloom filters, much smaller cache and no compression